### ISSUE TITLE
PHP 8.2 support

### ISSUE THREAD

**Issue by jadeops** [catfan/Medoo#1070]
Medoo 2.1.7 | PHP 8.2 | MySQL | Linux

## Deprecations

```
PHP Deprecated:  Using ${var} in strings is deprecated, use {$var} instead in /vendor/catfan/medoo/src/Medoo.php on line 870
PHP Deprecated:  Using ${var} in strings is deprecated, use {$var} instead in/vendor/catfan/medoo/src/Medoo.php on line 1321
```

**Comment by Machou** [comment]
Too

**Comment by catfan** [comment]
Merged #1072
